File: exo_10.rb
print "En quelle année es-tu né(e) ?\n> "
year = gets.chomp.to_i
x = 0

if year.zero?
  abort("Et ton année de naissance ?\n")

else if year == Time.new.year
  abort("Tu es trop jeune !\n")

end
end

while year <= Time.new.year
  puts x
  year += 1
  x += 1
end

File: exo_10.exs
year = IO.gets("En quelle année es-tu né(e) ?\n> ") |> String.trim("\n")
i = String.to_integer(year)
date = DateTime.utc_now() |> Map.fetch!(:year)
delta = date - i

cond do
  String.length(year) == 0 ->
    IO.puts("Et ton année de naissance ?\n")

  true ->
    for x <- 0..delta, i <= date, do: IO.puts(x)
end

